dist/Kotlin.JSON-tmLanguage

```diff
@@ -32,8 +30,8 @@
           ]
         },
         {
-          "match": "@[a-zA-Z_]\\w*",
-          "name": "meta.annotation.kotlin"
+          "match": "@(?:(?:file|property|field|get|set|receiver|param|setparam|delegate)\\s*:\\s*)?[a-zA-Z_]\\w*",
+          "name": "storage.type.annotation.kotlin"
```

Comment:
I've left a comment for discussion here https://github.com/nishtahir/language-kotlin/issues/42#issuecomment-779318939. That would need to be addressed before this gets merged

Reply:
Any updates? 🤔